__tests__/components/fileds/DateField.spec.js

```diff
@@ -0,0 +1,67 @@
+import DateField from '../../../src/components/fields/DateField.vue';
+import UI from '../../../src/store/ui';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import { restore } from 'sinon';
+import { register, unregister } from 'timezone-mock';
+import Vuex from 'vuex';
+
+const setupStoreMount = () => {
+	const uiModule = {
+		namespaced: true,
+		getters: {
+			getNameByKey: UI.getters.getNameByKey
+		}
+	};
+
+	const store = new Vuex.Store({
+		modules: {
+			ui:uiModule
+		}
+	});
+
+	const propsData = {
+		timestamp: 1615853185,
+		isShowTimestamp: true
+	};
+
+	return shallowMount(DateField, {
+		store,
+		localVue,
+		propsData
+	});
+};
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+describe('DateField component', () => {
+	// Arrange:
+	const timestamp = 1615853185;
+	let wrapper = {};
+
+	beforeEach(() => {
+		register('Etc/GMT-8');
+		wrapper = setupStoreMount();
+	});
+
+	afterEach(() => {
+		restore();
+		unregister();
+	});
+
+	it('processes props data', ()=> {
+		// Assert:
+		expect(wrapper.vm.timestamp).toBe(timestamp);
+		expect(wrapper.vm.isShowTimestamp).toBe(true);
+	});
+
```

Comment:
unsure what this test is adding

Reply:
is there any case you can think of where this test would fail but following test would not?
in other words, isn't following test already implicitly testing that everything is passed down correctly?

Reply:
yeah, agree, I got it now.
I will add 1 more test case with `shouldShowTimestamp : false`